doc/quickstart.rst

```diff
@@ -18,10 +18,249 @@ examples below:
+	   are of the stream.  You can see what output unit options are by passing a random string here
+	   and looking at the error message.
+	 - window (str):
+	   The aggregation period for showing when there is data in a stream.  You can pass a value like
+	   'days', 'hours', 'weeks', 'months'.  This will be given to TimeSelectViewer so any valid value
+	   there is acceptable here.  Defaults to days.
+	 - mdo (list(float)):
+	   An optional MDO to use to convert the data stored in this stream.  If passed this must be 3
+	   float values that define a linear transformation of the data stored in the stream. The values
+	   are interpreted as a m / d * value + o where the argument is specified as [m, d, o].
+	 - stream (str):
+	   A unique identifier for the stream that we wish to download and display.  This can be anything
+	   that could be passed to AnalysisGroup.fetch_stream including a 4-character variable ID, a
+	   partial string that matches the label of the stream, etc.
+
+
+There are two components of the help information for a LiveReport.  The first is just general information
+about what to expect from this report.  The second are any parameters that you can pass in order to
+adjust the report's output.  Not all reports have parameters and most parameters have sane defaults so it
+is safe to not pass anything.
+
+If you want to set a parameter you do it by passing an argument on the command line to ``analytics-host``
+of ``-a <parameter_name>=<parameter_value>``.  The parameter is automatically parsed and converted to
+the appropriate units as listed in the help text for the LiveReport.  For example, this report takes 
+an optional custom linear transformation parameter named ``mdo`` that is specified to be a list of 
+3 floating point numbers.  
+
+You would pass that on the command line as a string such as ``-a mdo=[1.0,2.0,3.0]``.  If you need to
+include a space in the parameter value, make sure to quote it so that the shell does not misinterpret
+the value as another argument to the program.
+
+Saving Data Offline
+-------------------
+
+Any LiveReport can be generated either directly from iotile.cloud by passing the ``slug`` of the device
+or archive that you want to use.  A slug is an alphanumeric identifier that starts with ``d--`` for 
+a device and ``b--`` for an archive.  
+
+If you have data that you wish to download and save, you can use the ``save_hdf5`` report to cache all
+data offline.
+
+.. note::
+
+	You will need to login with your iotile.cloud email address and password in order to download data
+	from iotile.cloud and you can only access data that belongs to devices or archives that you have 
+	access to.
+
+
+For example, let's say you want to download data from device ``d--abcd-0000-5678-1234`` and save it to
+a file named ``my-device.hdf5``  You would do:
+
+.. code:: bash
+
+	$ analytics-host -t save_hdf5 d--abcd-0000-5678-1234 -o my-device -c
+	Please enter your IOTile.cloud email: user@your-email.com
```

Comment:
Is there an option to define the email on the command line? e.g. -u/--user

Reply:
Yeah there is, list of all options is here:
https://github.com/iotile/iotile_analytics/blob/master/iotile_analytics_interactive/iotile_analytics/interactive/scripts/analytics_host.py#L26